src/auth/epics/index.js

```diff
@@ -42,10 +56,36 @@ export function authEpicFactory() {
             switchMap(action =>
                 authApi
                     .loginUser(action.payload)
-                    .then(actions.loginUserFulfilled)
+                    .then(() => {
+                        localStorage.setItem('authenticated', 'true');
```

Comment:
I think that isAuthenticated should be in redux. Probably in `auth` state

Reply:
yup I think so, but for that I have to handle this `redux-persist` think